src/app.rs

```diff
@@ -1583,6 +1599,10 @@ impl Window {
                     }
                 });
         });
+        ui.checkbox(
+            &mut self.config.search_state.whole_word,
+            "whole-word matches only",
```

Comment:
To match the style of other UI buttons, how about something like:

```
Match whole words only
```

Reply:
https://github.com/StanfordLegion/prof-viewer/pull/32/commits/276f1c360062c239571d0ae5c1c0c32257e84b9f